core/src/main/scala/org/apache/spark/rdd/RDDOperationScope.scala

```diff
@@ -79,7 +78,7 @@ private[spark] class RDDOperationScope(
  * An RDD scope tracks the series of operations that created a given RDD.
  */
 private[spark] object RDDOperationScope extends Logging {
-  private val jsonMapper = new ObjectMapper().registerModule(DefaultScalaModule)
```

Comment:
This was added by @andrewor14 in #5729 as part of the initial DAG visualization patch. Here, the only class that we serialize happens to be `RDDOperationScope`, so I think this removal is addressed by the explicit `@JsonProperty` annotations that I added above.

Let me quickly double-check that the handling of `Option` is correct, though.